docs/maintainers/README.md

```diff
@@ -7,17 +7,18 @@ Firstly, thank you for your willingness to help with Exercism. We believe that v
 ## Getting your track into shape
 
 There are three stages to getting your track ready:
-1) Understanding what needs doing
-2) Preparing your track to accept contributions
-3) Managing incoming contributions
+
+1. Understanding what needs doing
+2. Preparing your track to accept contributions
+3. Managing incoming contributions
 
 We have tried to provide comprehensive documentation on what needs doing, starting with this file, and linking out. If anything can be made clearer, we would really appreciate PRs to these docs, so that we can help each other have the most seamless experience.
 
 Once your track has been "boostrapped" you'll find a `languages/$LANG/README.md` file with a checklist of things to do. We recommend working through each one in turn. If your track has multiple maintainers we recommend scheduling a Slack conversation as a kick-off meeting and dividing up work and responsibilities. All maintainers will have been invited to Slack. If anyone isn't responding on Slack, please reach out to @ErikSchierboom on Slack, who can email them.
 
 ## New track structures
 
-The key thing to understand about v3 is that we are making a strong distinction between Concept Exercises and Practice Exercises. All existing exercises will become Practice Exericses - exercises where students can practice the ideas they are learning, and discuss them with a mentor. The new Concept Exercises will be written from scratch, each with the aim of teaching one or more programming concepts that are necessary to become fluent in a given language.
+The key thing to understand about v3 is that we are making a strong distinction between Concept Exercises and Practice Exercises. All existing exercises will become Practice exercises - exercises where students can practice the ideas they are learning, and discuss them with a mentor. The new Concept Exercises will be written from scratch, each with the aim of teaching one or more programming concepts that are necessary to become fluent in a given language.
```

Comment:
I think it should be captialized. I think a `Practice Exercise` is a "proper noun" in this context.

Reply:
```suggestion
The key thing to understand about v3 is that we are making a strong distinction between Concept Exercises and Practice Exercises. All existing exercises will become Practice Exercises - exercises where students can practice the ideas they are learning, and discuss them with a mentor. The new Concept Exercises will be written from scratch, each with the aim of teaching one or more programming concepts that are necessary to become fluent in a given language.
```